packages/velo-external-db-core/lib/converters/query_validator.js

```diff
@@ -15,6 +16,11 @@ class QueryValidator {
     
     }
     
+    validateGetById(fields, itemId) {
+        if (!itemId) throw new InvalidQuery('A value must be provided for itemId')
+        this.validateFilter(fields, getByIdFilterFor(itemId))
```

Comment:
this logic seems funny, we are validating a filter we synthetically create, i don't see the value in this. WDYT ?